irclib.py

```diff
@@ -1,4 +1,4 @@
-# Copyright (C) 1999--2002  Joel Rosdahl
+# Copyright (C) 1999--2002 Joel Rosdahl
```

Comment:
No need to change whitespace, just makes the diff harder to review, and breaks the blame.